Xamarin.MacDev/XamMacSdk.cs

```diff
@@ -301,5 +301,9 @@ bool CheckSupportsFeature (string feature)
 		public bool SupportsLinkPlatform {
 			get { return CheckSupportsFeature ("link-platform"); }
 		}
+
+		public bool SupportsHybridAOT {
+			get { return Version >= new MacOSXSdkVersion (3, 2, 0); }
```

Comment:
You're supposed to add a feature check (not a version check), like this: `CheckSupportsFeature ("link-platform");`.

This is where you want to add the feature: https://github.com/xamarin/xamarin-macios/blob/master/Versions-mac.plist.in

And that should be back-ported to all branches that support it.